src/Mutator/FunctionSignature/ProtectedVisibility.php

```diff
@@ -114,8 +115,8 @@ public function canMutate(Node $node): bool
 
     private function hasSameProtectedParentMethod(Node\Stmt\ClassMethod $node): bool
     {
-        /** @var ClassReflection $reflection */
         $reflection = $node->getAttribute(ReflectionVisitor::REFLECTION_CLASS_KEY);
+        Assert::isInstanceOf($reflection, ClassReflection::class);
```

Comment:
I think it would make more sense for `ReflectionVisitor` to provide a utility instead:

```suggestion
        $reflection = ReflectionVisitor::getReflectionClass($node);
```

```php
function getReflectionClass(Node $node): ClassReflection {
  $reflection = $node->getAttribute(self::REFLECTION_CLASS_KEY);
  Assert::isInstanceOf($reflection, ClassReflection::class);

  return $reflection;
}
```

I see that sometimes we want a nullable variant, to which we could have `::tryToGetReflectionClass()`

Reply:
added `ReflectionVisitor::getReflectionClass` - thx